CMakeLists.txt

```diff
@@ -131,15 +125,11 @@ endif()
 
 init_code_checks()
 
-#Turn on rpath stuff
 include(SetRpath)
 
-#Build pelec executables and link to amrex library
 add_subdirectory(Exec)
 
-if(PELEC_ENABLE_TESTS)
```

Comment:
Do we just always have testing on now? Fine by me

Reply:
Yeah, I can't say I've ever turned it off and since CMake is mostly for testing the code, where I use GNUmake for running it, it seemed like we should always have testing on.